enhanced/generate/main.go

```diff
@@ -54,6 +55,29 @@ func (m Metric) ConstLabels() map[string]string {
 	return nil
 }
 
+func (m Metric) Unit() float64 {
+	if strings.Contains(m.Help, "kilobytes") {
+		if strings.HasPrefix(m.FqName(), "node_") {
+			excluded := map[string]struct{}{
+				"in":    {},
```

Comment:
is it enough to set `in` or it is needed to put `pswpin` here to exclude mertic convertion?

Reply:
The metric here is called `swap_in`. It's just generator, ugly as hell at this point but does it job. Always verify the list of metrics, this is used on runtime.